python_bot.py

```diff
@@ -33,6 +33,11 @@
 discordApiKey = config.bot_token #from config.py file
 giphyApiKey = config.giphy_api_key #from config.py file
 
+# Update for each revision using format yyyy-mm-dd_#
+# where '#' is the release number for that day.
+# e.g. 2019-03-31_1 is the first relase of March 1st, 2019
+version = '2019-03-01_1'
```

Comment:
If we are going to use a global variable we should put it near the top of the file.

Reply:
it is, should I mark it as global. it is right underneath the other globals

Reply:
Oh. The git diff bamboozled me. I'm still a little fuzzy on how global works, but I believe it only needs to be marked global if it's not constant. You tested it in the discord then it should work as is. https://www.geeksforgeeks.org/global-keyword-in-python/